### ISSUE TITLE
Gentoo ebuild missing dependency on media-gfx/qrencode

### ISSUE THREAD

**Issue by Hawk777** [EionRobb/purple-discord#414]
The plugin now requires `qrencode` to compile, but it is not declared as a dependency in the Gentoo ebuild linked from the README.

**Comment by Hawk777** [comment]
It looks like it’s some kind of GitHub file or published artifact or something, not actually part of the Git repository, so I don’t think I can use a PR to change it. I’ve attached the new file here.

[ebuild.zip](https://github.com/EionRobb/purple-discord/files/9126517/ebuild.zip)

**Comment by heeplr** [comment]
@Hawk777 
That ebuild seems to be part of the [gentoo-instructions](https://github.com/EionRobb/purple-discord/tree/gentoo-instructions) branch. You could create a PR for that branch.

But besides that, the regular "gentoo way" would be to use an overlay (https://gpo.zugaina.org/x11-plugins/purple-discord) or even better, create a [pull request](https://github.com/gentoo/gentoo) to have the working ebuild added to the main tree.

**Comment by Hawk777** [comment]
> That ebuild seems to be part of the [gentoo-instructions](https://github.com/EionRobb/purple-discord/tree/gentoo-instructions) branch. You could create a PR for that branch.

I don’t think that makes sense. The head commit of that branch is in the ancestry of `master`, and a subsequent commit on `master`, ab31cc322e7a2b923518149d7ae304e565c086ce, removed the file from the repo and changed the README link to point at a GitHub file attachment rather than a file in the repo. So it seems like it *used* to be in the repo, and was intentionally moved out of the repo to a file attachment.

> But besides that, the regular "gentoo way" would be to use an overlay (https://gpo.zugaina.org/x11-plugins/purple-discord)

I already have it in my personal one, but I hadn’t planned on becoming a long-term maintainer myself, just fixing the immediate problem.

> create a [pull request](https://github.com/gentoo/gentoo) to have the working ebuild added to the main tree.

Likewise, plus the QA requirements are rather higher there; given that this is a live-only situation (there are no proper releases), I’m not sure whether they’d even want it upstream.

Feel free to do either of those things yourself if you like though!
